SPEC.md

```diff
@@ -288,6 +288,10 @@ Anything implementing this spec should enforce any signing rules set in place by
 
 Discovery URLs that require interpolation are [RFC6570](https://tools.ietf.org/html/rfc6570) URI templates.
 
+### Discovery Authentication
+
+Authentication during the discovery process is optional. If an attempt at fetching the discovery URL, an app container image, or signature returns a `401 Unauthorized` implementations should enforce any authentication rules set by the operator.
```

Comment:
OK, how about "implementations should enact the authentication policy set by the operator (for example, ...)